android-components/components/feature/search/src/main/java/mozilla/components/feature/search/telemetry/Utils.kt

```diff
@@ -113,28 +99,17 @@ private fun getTrackKeyFromCookies(
                 .split("=")
                 .map { item -> item.trim() }
 
-            if (valueList.size == 2 && valueList[0] == followOnCookie.codeParam &&
-                followOnCookie.codePrefixes.any { prefix ->
-                    valueList[1].startsWith(
-                        prefix,
-                    )
+            if (valueList.size == 2 && valueList[0] == followOnCookie.codeParamName &&
+                provider.taggedCodes.any { prefix ->
```

Comment:
I see this was `codePrefixes` -> `taggedCodes`. From the comments those sound like the same thing, just want to double-check that that's right.

Reply:
That is correct!